src/qml/MapCanvas.qml

```diff
@@ -60,106 +76,178 @@ Item {
     freeze: false
   }
 
-  PinchArea {
-    id: pinchArea
+//    TapHandler {
```

Comment:
@3nids , that should probably be taken away prior to merging.

Reply:
this is the code with the confirmed single click which is not used anymore.
But I realized that the double click is doing a zoom in AND an identify at the moment.

You were also suggesting to start the digitizing on a long press (and to finish on long press), @m-kuhn thoughts?